### DISCUSSION TITLE
Click/hover interaction events on track data

### DISCUSSION THREAD

**Discussion by cukelarter** [gosling-lang/gosling.js#1056] [Ideas]
Hi! I read the User Interaction documentation and was curious to see if there were any plans or workarounds to enable on-click or hover interactions with track data? For example, in the UCSC and other genome browsers you are able to click on portions of a track to bring up more information about that specific region, ie Ensemble id, rsid, coordinates, or even custom track fields. I've attached a basic example of the type of display I'm thinking of: 
![goslingexample](https://github.com/gosling-lang/gosling.js/assets/67754172/78b315e4-fe36-4fdb-bb5b-b36cdd69ad96)

**Comment by sehilyi** [comment]
Hi @cukelarter, thanks for asking question!

I think you are finding for "tooltip" (http://gosling-lang.org/docs/reference#tooltip). Any specified data in this property will appear in the tooltip upon _hovering_.

For instance, you can specify the following lines of code from the "Mark Displacement" example you provided


![Screenshot 2024-05-06 at 3 13 41 PM](https://github.com/gosling-lang/gosling.js/assets/9922882/039d64cc-adfc-490f-8901-7fe2454a8303)

which will display the following tooltip upon hovering.

![Screenshot 2024-05-06 at 3 13 44 PM](https://github.com/gosling-lang/gosling.js/assets/9922882/be928933-74be-44ca-af88-d7b9945f0b12)

**Reply by cukelarter** [reply]
thanks for pointing this out and providing an example, I was able to get it working! :)

**Reply by sehilyi** [reply]
Great! Please feel free to mark your question "answered"!
